HISTORY.md

```diff
@@ -1,107 +1,159 @@
-        * Add missing type: `MARTYR` & fix wrong specified types in brazil calendar.
-
-## 1.2.3 / 2017-03-21
-* Added French translations for France, Canada and Belgium.
-* Fix issue where national feasts appear more than once in the calendar output
-
-## 1.2.2 / 2017-03-21
-* Merge PRs containing bug fixes and improvements, thanks [Etienne Magnier](https://github.com/emagnier) and [Damiaan Dufaux](https://github.com/Dev1an)
-
-## 1.2.1
-* Added caching mechanism for common celebration dates in the liturgical calendar to improve calendar processing output performance. Added more titles metadata to national and general celebrations.
-
-## 1.2.0
-* Major rewrite for better extensibility and functionality. All previous revisions have been marked for deprecation in favor of this new rewrite.
+## 1.3.0 /2020-01-25 <a name="1.3.0"></a>
+
+romcal changed a lot since [1.2.4](#1.2.4), therefore we list here only the main changes and small changes were combined. For more thorough list of changes, check the release notes for [1.3.0-beta.1](#1.3.0-beta.1), [1.3.0-beta.2](#1.3.0-beta.2), [1.3.0-beta.3](#1.3.0-beta.3), [1.3.0-rc.1](#1.3.0-rc.1) and [1.3.0-rc.2](#1.3.0-rc.2).
+
+### General changes
+
+- Revamp the code to use ES6 syntax.
+- Drop support for Node v4, v5.
+- Integrated module with `std/esm` to read `.mjs` files and `babel` to ensure compatibility and seamless usage with lower version of node and requiring via CommonJS.
+- Expose constants and calendar helper functions and via module exports.
+- Refactor rank types of celebrations ([issue #109](https://github.com/pejulian/romcal/issues/109)).
+
+### New features
+
+- Introduce the new `drop` property that allows national calendars to omit unwanted celebrations from their output ([issue #14](https://github.com/pejulian/romcal/issues/14)).
+- Introduce new `christmastideIncludesTheSeasonOfEpiphany` flag to allow output to include or exclude the "season" of Epiphany from being merged into Christmastide.
+
+### Calendar and celebrations
+
+- Added national calendars for Czech Republic and Slovakia.
+- Drop Shrove Monday and Tuesday from the General Roman Calendar ([issue #90](https://github.com/pejulian/romcal/issues/90)).
+- Epiphany is always celebrated on Jan 6 in Slovakia ([issue #29](https://github.com/pejulian/romcal/issues/29)).
+- Apply changes in the General Roman Calendar made by the Holy See.
+
+### Localization
+
+- Add Slovak locale files.
+- Some localization keys where renamed in order to confirm with official English names of the celebrations.
+
+### Build process
+
+- Update and simplify the build process.
+- Add automated build process to transpile the code after installing and before publishing.
+- Distribution files are now not included anymore in the codebase.
+- In addition to the `romcal.bundle.min.js` file (for browsers), source code is also transpiled by Babel in the `dist/` directory. This makes it easier to include romcal in any node.js package.
```

Comment:
...is also transpiled by Babel and Webpack...

~~This makes it easier to include romcal in any node.js package.~~ That new feature is to allow using romcal directly in the browser (which is new), so not only in a node.js package or environment (babel and webpack allow to have the same codebase for backend/node.js and frontend/browser).

Reply:
What do you think about the following?

> - In addition to the `romcal.bundle.min.js` file (for browsers), source code is also transpiled by Babel and Webpack in the `dist/` directory. This feature makes it possible to use romcal directly in the browser, so not only in a node.js package or environment (babel and webpack allow to have the same codebase for backend/node.js and frontend/browser).